src/js/models/cpu_list_model.js

```diff
@@ -173,6 +173,13 @@ YUI.add('cpu-list-model', function(Y) {
                         var cpuData = item.cpuData; // the N CPUs, M values per CPU
                         var readTime = item.readTime;
 
+                        // just in case there is somehow a duplicate of KG-14770
+                        // that shows up for System data, put in a check that
+                        // cpuData is actually non-null.
+                        if (cpuData == null || cpuData == undefined) {
```

Comment:
Looks like if cpuData is undefined then "cpuData == null" will return true, so this code is fine